packages/components/app/styles/components/dialog.scss

```diff
@@ -0,0 +1,123 @@
+//
+// DIALOG
+//
+
+@use "../mixins/focus-ring" as *;
+
+.hds-dialog {
+  flex-direction: column;
+  padding: 0;
+  background: var(--token-color-surface-primary);
+  border: 1px solid rgba(59, 61, 69, 40%); // --token-color-border-strong;
+  border-radius: 10px;
+  box-shadow: var(--token-elevation-higher-box-shadow);
```

Comment:
this needs to be discussed with the designers, because we have an `overlay` style already predefined (in Figma as well in code) but if this is the one we are using for something which is an "overlay" them we need to understand if that needs to remain (in which case would it be used then?) or we can remove it

/cc @heatherlarsen @jorytindall

Reply:
Yes, we have discussed this, there are couple of options that I presented:

1. Use the `overlay` token as it currently exists
2. Use the `overlay` token but update it a bit. It was created before really being used in context so might not really fit our needs.
3. Use the `higher` token. There are issues with this, the usage and semantic isn't really correct.
4. Create a whole new token for the dialog/modal. This seems unreasonable at this time.

I believe I had raised the question in review but there wasn't a clear answer. I'm an advocate for **option 2** and will raise it in review.

Reply:
Rebased after #646 and updated to `--token-surface-overlay-box-shadow` – @jorytindall let me know if I got it wrong though.

Reply:
Nailed it 👍